.travis.yml

```diff
@@ -126,6 +126,26 @@ before_install:
           (cd php-$MIN_PHP && ./configure --enable-sigchild --enable-pcntl && make -j2)
       fi
 
+    - |
+      # Install ext/memcached from source
```

Comment:
yes that's true. Maybe it's even better to compile the .so files once and download them instead of compiling (until a release is made) because these are master branches ...
I would first like to get to the point where 7.3 is working. Any idea to fix the OPT_LIBKETAMA_COMPATIBLE issue? I am a bit lost

Reply:
Memcached should be in its way before the end of the year.

https://github.com/php-memcached-dev/php-memcached/issues/408#issuecomment-448700071

Reply:
okay. Trying to isolate the memcache tests now ...